[doc] Improve documentation for binaries

woshilapin opened the pull request:
Ref. ND-929

datanel commented on gtfs2netexfr/src/main.rs:
```diff
@@ -30,40 +30,41 @@ struct Opt {
     input: PathBuf,
 
-    /// Output directory
+    /// Output directory.
```
Strange behaviour. All the dots are not displayed. Only if there are multilines.

woshilapin replied:
Ah, indeed, I didn't notice and that is strange. Although, I would let them for at least consistency in the source file.

woshilapin commented:
> The point of this PR as I understood it was to have all sub-readme very similars.
> 
> Any diff with gtfs2netexfr/README.md should be just about specificities of the bin:
> 
>     * If using proj, the topo should be the same (no mention of the version, just in main readme)
> 
>     * adding/removing specific required params
> 
>     * same topo on the NeTEx manual-check (link will change with #665 : rebase!)
> 
>     * Maybe some other diffs I didn't spot for `restrict` or `ntfs2ntfs` ?
> 
> 
> So current work is good for me, but I think there is a little more changes to do on sub-readmes.

So, here is the last modifications I made:

- I remove the section about configuration in `gtfs2ntfs/README.md` (it is already covered by the `--help` option
- I removed the PROJ version from `ntfs2netexfr` (already mentioned in `CONTRIBUTING.md`)
- I added a link to the section `check the output manually` in `ntfs2netexfr` (and I rebased for that)
